src/overlays/actors/ovl_En_Minislime/z_en_minislime.h

```diff
@@ -5,9 +5,49 @@
 
 struct EnMinislime;
 
+typedef void (*EnMinislimeActionFunc)(struct EnMinislime*, GlobalContext*);
+
+#define MINISLIME_NUM_ACTOR 15
```

Comment:
What is this?

Reply:
When Bigslime breaks apart, 15 separate minislime actors spawn. A lot of the minislime functionality is controlled in Bigslime, including everything involving draw. Bigslime stores a pointer to each of the 15 actor instances of minislime in the struct `/* 0x0A14 */ EnMinislime* minislime[MINISLIME_NUM_ACTOR];` from bigslime.h, and there are 11 separate for-loop `for (i = 0; i < MINISLIME_NUM_ACTOR; i++)` in Bigslime doing some sort of operation on all instances of minislimes. Does that make sense?

Reply:
Perhaps `#define MINISLIME_SPAWN_NUM`?